### ISSUE TITLE
What about app icon?

### ISSUE THREAD

**Issue by IamR3m** [iamr0s/Dhizuku#61]
Why don't you create your own icon and use the default one?

**Comment by SunWuyuan** [comment]
If Shizuku doesn't have any restrictions on icons Changing the color scheme of their icons is a good idea (laughs)

**Comment by SunWuyuan** [comment]
![image](https://github.com/iamr0s/Dhizuku/assets/88357633/9b271d77-b265-4b88-a1e7-1311c7ea39b4)

**Comment by shuvashish76** [comment]
Duplicate #55

**Comment by lutianyu2001** [comment]
> If shizuku doesn't have any restrictions on icons, then changing the color tone of their icons would be a good idea (laughs

Good idea. 😂 Unfortunately Shizuku seems have extremely strict regulation on the usage of their icon, so it's impossible to recreate on their icon.

**Comment by lutianyu2001** [comment]
I've created an icon initially only for my personal use, but I think generally it is good to be released, so I created a PR https://github.com/iamr0s/Dhizuku/pull/93 for it. 😉

**Comment by s1204IT** [comment]
Some of the conditions for the icon are 
- That it is not similar to Shizuku
 and 
- That it is simple and original like InstallerX.

**Comment by Sunwuyuan** [comment]
maybe we can have communicate with  shizuku about re use its icon?

**Comment by s1204IT** [comment]
This is more a request from R0S himself than a desire to avoid competition with Rikka.

**Comment by Ermageeerd** [comment]
Here's a conceptual icon. It's inspired by Shizuku's icon, but different enough to work. ![Image](https://github.com/user-attachments/assets/c72099cc-a259-4f4b-a0da-4ad6e3b3ee56)

**Comment by Ermageeerd** [comment]
(note: it is more of a "concept" icon, if you like it, tell me and I can try to make it production ready.)

**Comment by s1204IT** [comment]
Completed! #129
